lib/aerosol/instance.rb

```diff
@@ -22,6 +22,14 @@ def private_ip_address
     description[:private_ip_address]
   end
 
+  def address
```

Comment:
Do we want to fallback to the private ip or the public hostname? Or should it be config?